fe/fe-core/src/main/java/org/apache/doris/mysql/privilege/RoleManager.java

```diff
@@ -211,6 +212,31 @@ public void getRoleInfo(List<List<String>> results) {
         }
     }
 
+    public void getRoleWorkloadGroupPrivs(List<List<String>> result, Set<String> limitedRole) {
+        for (Role role : roles.values()) {
+            if (ClusterNamespace.getNameFromFullName(role.getRoleName()).startsWith(DEFAULT_ROLE_PREFIX)) {
+                continue;
+            }
+
+            if (limitedRole != null && !limitedRole.contains(role.getRoleName())) {
+                continue;
+            }
+            String isGrantable = Auth.isAdminPriv(role.getGlobalPrivTable()) ? "YES" : "NO";
```

Comment:
role.checkGlobalPriv(wanted) instaed？

Reply:
```checkGlobalPriv``` can only check user is admin, but it can not check whether a role has granted privilege.